docs/manual/must-call-checker.tex

```diff
@@ -0,0 +1,204 @@
+\htmlhr
+\chapterAndLabel{Must Call Checker}{must-call-checker}
+
+The Must Call Checker conservatively over-approximates
+the set of methods that an object should call before it is de-allocated.
+It is intended to be run as a subchecker of another checker: on its own, it does not
```

Comment:
How would it be used by another checker?

Reply:
The eventual client has three stages:
1. run the Must Call Checker to over-approximate the methods that must be called
2. run the Called Methods Checker to under-approximate the methods that actually were called
3. check that 2 is a superset of 1 when an object goes out of scope.

This PR only provides part 1. Part 2 already exists (the Called Methods Checker). Another PR will later add part 3.

Reply:
Thanks for the additional context.  Do you expect checkers beside the one mentioned in 3 above to use the MustCall checker?

Reply:
Not at this time.